roles/route-server-docker-image/templates/build-routes.sh.j2

```diff
@@ -124,6 +131,44 @@ function buildOtpGraphHSL() {
   java -Xmx8G -jar $OTP_JAR --build hsl/ 2>&1 | tee ${ROOT}/hsl/Graph.build.log.txt
 }
 
+#add (or modify) feed_info.txt that contains the feed_id
+function add_feed_id() {
+  set +o pipefail
+  filename=$1
+  id=$2
+
+  contains_fileinfo=`unzip -l $filename|grep feed_info.txt|wc -l`
+
+  if [ "$contains_fileinfo" -ne "1" ]; then
+     echo "creating new feed-info"
+     #no feed info available in zip, write whole file
+     cat <<EOT > feed_info.txt
+feed_publisher_name,feed_publisher_url,feed_lang,feed_id
+$id-fake-name,$id-fake-url,$id-fake-lang,$id
+EOT
+  else
+     unzip -o $filename feed_info.txt
+     #see if if feed_id is already there
+     count=`grep feed_id feed_info.txt|wc -l`
+     if [ "$count" -ne "1" ]; then
+       echo "adding feed_id column"
+       #no feed_id in feed_info.txt, append
+       awk -vc="feed_id" -vd="$id" 'NR==1{$0=c","$0}NR!=1{$0=d","$0}1' feed_info.txt > feed_info_new.txt
+       #todo rename
```

Comment:
This is done [here](https://github.com/HSLdevcom/digitransit-deploy/pull/76/files#diff-b0f7982957222a60451e58ae237d6a98R164) now?